components/package.json

```diff
@@ -32,7 +32,7 @@
     "jquery.cookie": "1.4.1",
     "jquery.flot.tooltip": "^0.9.0",
     "jquery.hotkeys": "jeresig/jquery.hotkeys#master",
-    "jszip": "^3.9.0",
+    "jszip": "^3.9.1",
```

Comment:
I have no idea why this is marked as a change. `jszip` is at version 3.9.1 in `dev` as well